package.json

```diff
@@ -62,7 +62,7 @@
     "livestamp": "github:mattbradley/livestampjs#7572242",
     "moment": "^2.18.1",
     "turbolinks": "^5.0.3",
-    "tagging": "^0.0.4-a"
+    "tagging": "thesecretmaster/tagging#master"
```

Comment:
If this is a GH reference, it probably needs to be prefixed with `github:` - see L62.

Reply:
It works. I can add the prefix, but I pulled that from npms docs.